testing/test_aux.sh

```diff
@@ -191,4 +195,65 @@ head inst/run-port-*/finish/nmap*/*
 
 tcpdump -en -r inst/run-port-01/scans/test_nmap.pcap icmp or arp
 
+
+function monitor_interface {
+    while true; do 
+        found=$(cat inst/cmdrun.log 2>/dev/null | grep "$2")
+        if [ -n "$found" ]; then
+            echo found $2
+            eval $1 
+            break
+        fi
+        test_done=$(cat $TEST_RESULTS | grep "Done with tests")
+        if [ -n "$test_done" ]; then
+            break
+        fi
+    done &
+}
+rm -f inst/cmdrun.log
```

Comment:
probably want to put this inside of monitor_interface so it gets cleared each time (otherwise there might be cruft between invocations)